src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,59 @@
+package calculator;
+
+import java.util.Objects;
+
+public class StringCalculator {
+    public static int cal(String input) {
+
+        if (isBlank(input)) {
+            throw new IllegalArgumentException("빈 문자가 입력되었습니다.");
+        }
+
+        String[] stringList = split(input);
+        int stringLength = stringList.length;
+        int result = parseInteger(stringList[0]);
+
+        if (stringLength == 1) {
+            return result;
+        }
+
+        for (int i=2; i<stringLength; i = i+2) {
+            result = calculator(result, parseInteger(stringList[i]), stringList[i-1]);
+        }
+
+        return result;
+    }
+
+    private static boolean isBlank(String input) {
+        return input == null || input.isBlank();
+    }
+
+    private static String[] split(String input) {
+        return input.split(" ");
+    }
+
+    private static int calculator(int result, int number, String operation) {
+        if (Objects.equals(operation, "+")) {
+            return Math.addExact(result, number);
+        }
```

Comment:
```suggestion
    private static int calculator(Number result, Number number, String operation) {
        if (Objects.equals(operation, "+")) {
            return result.plus(Number);
        }
```

`Number`라는 객체가 있으면 사칙연산을 객체에게 시켜 볼수 있을것 같아요!

Reply:
> 나눗셈의 경우 결과 값이 정수가 아닐 때 Error를 뱉어야 되나요?(현재는 미구현 상태)

`Number`라는 정수만을 가지는 객체로 만들어본다면
정수가 아닌 경우가 발생하지는 않을것?? 같아요!

Reply:
>  음수도 나올 수 있도록 구현되어야 되나요?(현재는 구현된 상태)

정수니까 음수도 가능할것 같아요! :)